src/timeline/js/directives/clip.js

```diff
@@ -421,8 +407,9 @@ App.directive('tlMultiSelectable', function(){
 						item = findElement(scope.project.effects, "id", id);
 					}
 					
-					if (scope.Qt)
+					if (scope.Qt) {
```

Comment:
Ugh, _THANK YOU_. Conditional blocks without enclosing braces are just bug magnets, they're practically _taunting_ anyone who looks at the code. "Try to add another instruction to me, screw up the program logic!"

Reply:
@ferdnyc - Yep, its a personal preference of my own too. I HATE when people do single line if statements. Its just not consistent with the rest of them and yeah it can be a magnet for future bugs.